buildSrc/build.gradle.kts

```diff
@@ -26,7 +26,6 @@ repositories {
 }
 
 dependencies {
-  implementation(gradleApi())
   implementation(libs.aspectj.postCompileWeaving)
   implementation(libs.checkerFramework)
   implementation(libs.jetbrains.intellij)
```

Comment:
If it comes to the renames... does this rename change the logic/classpath in any way? Is it more aligned with some Gradle conventions? 🤔

Reply:
Is it actually related to the removal of `gradleApi()` above?

Reply:
More aligned with gradle conventions: [Gradle docs](https://docs.gradle.org/current/userguide/custom_plugins.html#sec:precompiled_plugins)

Seems that gradleApi is just not needed.

Reply:
> More aligned with gradle conventions: [Gradle docs](https://docs.gradle.org/current/userguide/custom_plugins.html#sec:precompiled_plugins)

Hmm does the code in `buildSrc/src/...` actually contain any precompiled _plugins_? this is what this docs seems to refer to